log/lvl_test.go

```diff
@@ -212,9 +211,9 @@ func ExampleLvlf1() {
 	OutputToBuf()
 
 	// Output:
-	// 1 : (                        log.ExampleLvlf1:   0) - Before
-	// 1 : (log.thisIsAVeryLongFunctionNameThatWillOverflow:   0) - Overflow
-	// 1 : (                        log.ExampleLvlf1:   0) - After
+	// 1 :                             fake_name.go:0 - Before
+	// 1 :                             fake_name.go:0 - Overflow
```

Comment:
Because the function name doesn't to find where the log is, where the file path allows IDEs to navigate to the exact location quickly. I'm kinda tired of Ctrl+F to find stuff..

Reply:
Can we have the cake and eat it too? Mean: can we have the function-name, plus the method name? I often look at the method name in the debug-output...

Reply:
Yes, we can!

Reply:
OK, looked at it again, and it's either/or. Which is fine with me, but I'd use the `ParseEnv` to enable it on the command line...